### PR TITLE
Moving Bakame CSV collection bridge to league\csv main repository

### PR DESCRIPTION
**Pull request by nyamsprod**
Moving [Bakame CSV collection bridge](https://github.com/bakame-php/csv-doctrine-collections-bridge) to League\CSV main repository as a READONLY extension package.

Using league\flysystem strategy to create READONLY packages

@frankdejonge a quick review to see if I did not forget something would be nice 🎐 🙏🏾

### CODE REVIEW

File: .github/workflows/publish-subsplit.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+                  user_email: 'nyamsprod@gmail.com'
+            - name: Cache splitsh-lite
+              id: splitsh-cache
+              uses: actions/cache@v2
+              with:
+                  path: './.splitsh'
+                  key: '${{ runner.os }}-splitsh'
+            - uses: frankdejonge/use-subsplit-publish@1.0.0-beta.5
+              with:
+                  source-branch: '3.x'
```
Comment (frankdejonge) [inline .github/workflows/publish-subsplit.yml:36]: Is this correct?

Comment (nyamsprod) [reply .github/workflows/publish-subsplit.yml:36]: good catch I did not update that part 👍🏾

Comment (nyamsprod) [reply .github/workflows/publish-subsplit.yml:36]: should be `master` instead if I am correct

**Comment by nyamsprod** [comment]
@frankdejonge  After merging I get this error
>/usr/bin/git push csv-doctrine 545304857800875fffff38d0ee59aa41613392e2:refs/heads/master -f
remote: Invalid username or password.
fatal: Authentication failed for 'https://github.com/thephpleague/csv-doctrine.git/'
Error: Error: The process '/usr/bin/git' failed with exit code 128
##[debug]Node Action run completed with exit code 1
##[debug]Finishing: Run frankdejonge/use-subsplit-publish@1.0.0-beta.5

What did I do wrong this time 🤔 . I believe the secret token is not being read 🤷🏾‍♂️

**Comment by nyamsprod** [comment]
Found the issue
Fixed the issue 
Subsplit worked like a charm 💯

**Pull request merged**